plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergDistributed.java

```diff
@@ -1,102 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.iceberg;
-
-import com.google.common.collect.ImmutableMap;
-import io.trino.testing.AbstractTestDistributedQueries;
-import io.trino.testing.QueryRunner;
-import io.trino.testing.sql.TestTable;
-import org.testng.SkipException;
-
-import java.util.Optional;
-
-import static io.trino.plugin.iceberg.IcebergQueryRunner.createIcebergQueryRunner;
-
-public class TestIcebergDistributed
-        extends AbstractTestDistributedQueries
-{
-    @Override
-    protected QueryRunner createQueryRunner()
-            throws Exception
-    {
-        return createIcebergQueryRunner(ImmutableMap.of(), REQUIRED_TPCH_TABLES);
-    }
-
-    @Override
-    protected boolean supportsViews()
-    {
-        return false;
-    }
-
-    @Override
-    protected boolean supportsCommentOnColumn()
-    {
-        return false;
-    }
-
-    @Override
-    protected TestTable createTableWithDefaultColumns()
-    {
-        throw new SkipException("Iceberg connector does not support column default values");
-    }
-
-    @Override
-    public void testDelete()
-    {
-        // TODO (https://github.com/trinodb/trino/pull/4639#issuecomment-700737583)
-    }
-
-    @Override
-    public void testRenameTable()
-    {
-        assertQueryFails("ALTER TABLE orders RENAME TO rename_orders", "Rename not supported for Iceberg tables");
-    }
-
-    @Override
-    protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMappingTestSetup dataMappingTestSetup)
-    {
-        String typeName = dataMappingTestSetup.getTrinoTypeName();
-        if (typeName.equals("tinyint")
-                || typeName.equals("smallint")
-                || typeName.startsWith("char(")) {
-            return Optional.of(dataMappingTestSetup.asUnsupported());
-        }
-
-        if (typeName.startsWith("decimal(")
-                || typeName.equals("time")) {
-            // TODO this should either work or fail cleanly
-            return Optional.empty();
-        }
-
-        if (typeName.equals("timestamp")) {
-            return Optional.of(new DataMappingTestSetup("timestamp(6)", "TIMESTAMP '2020-02-12 15:03:00'", "TIMESTAMP '2199-12-31 23:59:59.999999'"));
```

Comment:
I would squash this commit with the previous one. 
You can mention the fact you're mering TestIcebergDistributed into the other test class somewhere in the commit message. This way, it would be possible to find out where `filterDataMappingSmokeTestData` (the remaining copy, not this one) came from